org.rphys.srb2.yml

```diff
@@ -0,0 +1,101 @@
+app-id: org.rphys.srb2
```

Comment:
`rphys.org` doesn't seem resolve anywhere. See https://docs.flathub.org/docs/for-app-authors/requirements#application-id

> The Application ID should be a real URL of a domain that the app author has control over or where their app is hosted.

Reply:
I changed the app name to use github.com. Should I open a new PR with the branch name fixed or keep using this one?